addons/binding/org.openhab.binding.astro/README.md

```diff
@@ -101,10 +101,17 @@ An complementary altitude (optional) configuration item can also be specified to
     * **group** `apogee`
         * **event**: `APOGEE`
 
+### Channel config
 **Offsets:** For each event group you can optionally configure an offset in minutes.
 The offset must be configured in the channel properties for the corresponding thing.
 The minimum allowed offset is -1440 and the maximum allowed offset is 1440.
 
+**Earliest/Latest:** For each trigger channel and `start`, `end` datetime value, you can optionally configure the `earliest` and `latest` time of the day.  
+e.g sun#rise earliest=18:00, latest=20:00  
```

Comment:
if you intent to have new lines, please add an empty line inbetween (check how http://docs.openhab.org/addons/bindings/astro/readme.html#trigger-channels looks like)